EventGenerator/src/PionFilter_module.cc

```diff
@@ -0,0 +1,99 @@
+// Purpose: Event filter for RPC simulations
```

Comment:
This module should be split up into a filter module and an analyzer that produces the TTree.

Reply:
This means that the isNull parameter will not be needed.

Reply:
I will remove the ntuple, it was for debugging

Reply:
OK